electron/src/menu/developer.ts

```diff
@@ -37,12 +37,16 @@ const reloadTemplate: MenuItemConstructorOptions = {
   label: 'Reload',
 };
 
-const openDevTools = async (webViewIndex: number): Promise<void> => {
-  const snippet = `document.getElementsByTagName("webview")[${webViewIndex}].openDevTools({mode: "detach"})`;
-
+export const openDevTools = async (webViewIndex?: number): Promise<void> => {
   const primaryWindow = WindowManager.getPrimaryWindow();
+
   if (primaryWindow) {
-    await executeJavaScriptWithoutResult(snippet, primaryWindow.webContents);
+    if (typeof webViewIndex === 'number' && webViewIndex > 0) {
```

Comment:
```suggestion
    if (webViewIndex > 0) {
```

Reply:
`Object is possibly 'undefined' .ts(2532)`